nl/tested-judge/standalone/README.md

```diff
@@ -0,0 +1,170 @@
+---
+title: TESTed zonder Dodona
+description: "TESTed zonder Dodona"
+---
+
+# TESTed zonder Dodona
+Hoewel tijdens de ontwikkeling van TESTed er gefocust werd voor het gebruik op het Dodona-platform, kan TESTed ook los
+van Dodona gebruikt worden.
+In deze handleiding zullen we uitleg hoe je TESTed kan gebruiken los van het Dodona-platform.
+
+## 1. Klonen van de TESTed GitHub repository
+De eerst stap om TESTed los van Dodona te gebruiken is het klonen of downloaden van de
+[TESTed GitHub repository](https://github.com/dodona-edu/universal-judge).
+
+![Klonen TESTed GitHub repository](./klonen_tested_repo.png)
+
+## 2. Installeren software
+De volgende stap is het installeren van de afhankelijkheden van de TESTed judge.
+
+::: tip Windows 10 tip
+Windows 10 gebruikers raden we aan om het [Windows Subsystem for Linux](https://ubuntu.com/wsl) te gebruiken.
+:::
+
+### 1) Installeren kern software
```

Comment:
I would mention somewhere that this section is a relatively standard Python project (python & pip & requirements.txt)